pkg/relayer/miner/gen/template.go

```diff
@@ -15,7 +15,7 @@ package miner_test
 
 var (
 	// marshaledMinableRelaysHex are the hex encoded strings of serialized
-	// relayer.MinedRelays which have been pre-mined to difficulty 2 by
+	// relayer.MinedRelays which have been pre-mined to difficulty {{.difficultyBitsThreshold}} by
```

Comment:
Do we need to apply the same template on line 27?

Reply:
:roll_eyes::facepalm: - Thank you! :sweat_smile: